scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/SoftKeywords.scala

```diff
@@ -18,6 +18,7 @@ class SoftKeywords(dialect: Dialect) {
   object KwInfix extends IsWithName(dialect.allowInfixMods, "infix")
   object KwExtension extends IsWithName(dialect.allowExtensionMethods, "extension")
   object KwErased extends IsWithName(dialect.allowErasedDefs, "erased")
+  object KwTracked extends IsWithName(true, "tracked")
```

Comment:
@tgodzik already mentioned that you need to identify a dialect flag for this, and enable it for the appropriate dialect instances.

Reply:
Done, one thing to remember will be to add it to `3.5.0` once it is added. (Right now, I only added it to 3.future)